partridge/geo.py

```diff
@@ -0,0 +1,38 @@
+from typing import Dict, List
+
+import pandas as pd
+
+try:
+    import geopandas as gpd
+    from shapely.geometry import LineString, Point
+except ImportError as impexc:
+    print(impexc)
+    print("You must install GeoPandas to use this module.")
+    raise
+
+
+def build_shapes(df: pd.DataFrame) -> gpd.GeoDataFrame:
+    if df.empty:
+        return gpd.GeoDataFrame(
+            {"shape_id": [], "geometry": []}, crs={"init": "EPSG:4326"}
```

Comment:
Per @migurski's observation, this could be parameterized with this value as the default.

That would also help consolidate the crs dict and prevent typos, etc. Alternatively, moving this value to a constant and using that might also be safer than rewriting it everywhere.